test/util/diff-deep-equals.js

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {default as deepEquals} from 'deep-equal';
+import {default as deepEquals} from './deep-equals';
```

Comment:
What about giving this another name so others know the difference between this and normal equals?  Like `almost-deep-equals` or `approx-deep-equals` or `deep-equals-float`.  Google test does [`EXPECT_FLOAT_EQ`](https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#floating-point-comparison)

Reply:
Oh I would also consider adding a quick independent test to make sure deepEquals really does what we are expecting here, and if make any update it stays working like that.